source/contact.erb

```diff
@@ -0,0 +1,104 @@
+---
+title: "get in touch..."
```

Comment:
This title also shows in the page/tab title. This is also a key item for SEO so it probably should read something related to "Contact"

![screen shot 2015-12-18 at 12 11 20](https://cloud.githubusercontent.com/assets/885223/11896960/f9e3e7bc-a582-11e5-97c7-4a7583660486.png)